### ISSUE TITLE
#1066 - Not unique table/alias error only export

### ISSUE THREAD

**Issue by davidbouche** [phpmyadmin/phpmyadmin#13187]
I'm actually having an issue with PHPMyAdmin :

- I can execute my SQL query with nice result
- I get an error when I'm trying to export the same query

Here's the query :

```sql
SELECT pn.nomenclature_id as nomenclature_id, pn.lettre as lettre, a.reference as reference_article, at.nom as nom_article
FROM item a, item_translation at, item_nomenclature an, item_nomenclature pn, item p 
WHERE a.type = 1
AND a.id = at.id
AND at.lang = 'fr'
AND a.id = an.item_id
AND an.nomenclature_id = pn.nomenclature_id
AND pn.item_id = p.id
AND p.type = 2
AND p.marque_id = 2
```
Here's a video where I had the problem : https://youtu.be/Z5AAZhoX6W0

There's lot of thread on Not unique table/alias error but I didn't found any reason to explain why the query works in PHPMyAdmin "SQL" tab but not with export.

Thanks for your support, David.

### Server configuration
**Operating system**:
5.7.17-0ubuntu0.16.04.2-log - (Ubuntu)
**Web server:**
Apache/2.4.18 (Ubuntu)
**Database:**
libmysql - mysqlnd 5.0.12-dev - 20150407
**PHP version:**
7.0.15-0ubuntu0.16.04.4
**phpMyAdmin version:**
Version : 4.5.4.1deb2ubuntu2

### Client configuration
**Browser:**
Chromium Version 56.0.2924.76 Built on Ubuntu , running on Ubuntu 16.10 (64-bit)
**Operating system:**
Ubuntu 16.10 yakkety

**Comment by nijel** [comment]
There are certainly some problems with exporting queries with aliases, see https://github.com/phpmyadmin/phpmyadmin/issues/13222. I'm not sure if the root cause is same...

**Comment by dm79219** [comment]
Thanks @akashkumardce .  I was unable to use export due to this error. Was struck for a long time on this hampering my work. Your fix worked perfectly for me.

**Comment by williamdes** [comment]
Duplicate of #12241

**Comment by williamdes** [comment]
Will be fixed as #12241
